katello/hooks/pre_commit/13-hiera.rb

```diff
@@ -1,16 +1,43 @@
-TUNING_SIZES = ['default'] + ['medium', 'large', 'extra-large', 'extra-extra-large']
+# A mapping of tuning profile to the required CPU cores and memory in GB
+# TODO: check actual required values
+TUNING_SIZES = {
+  'default' => [1, 8],
```

Comment:
can this be a Hash instead of an Array? two values might still be possible to remember, but if we ever want to add more this will be messy.

also, how about modern style Hash syntax with `:` instead of `=>`?

Reply:
> can this be a Hash instead of an Array? two values might still be possible to remember, but if we ever want to add more this will be messy.

I considered that, but went with the simple route. If we add more (per scenario, disk space) then we can refactor it later

> also, how about modern style Hash syntax with `:` instead of `=>`?

We still need to support Ruby 2.0 and the key is a string anyway.

Reply:
I echo @evgeni , please make this a hash.

Reply:
I'll do so when we have a table of the actual required values so I'm not changing the same piece of code twice.

Reply:
I don't entirely follow what the actual required values has to do with it? I am thinking:

```
TUNING_SIZES = {
  'default' => {'cpu_cores' => 1, 'memory' => 8},
  'medium' => {'cpu_cores => 1, 'memory' => 10},
```

This is very readable for me without having to go read the code to figure out how these values are being used.

Reply:
The values I used for the profiles were placeholders. Luckily @sthirugn documented them in https://github.com/theforeman/theforeman.org/pull/1504 which makes my life easier :) Now that I have the correct list, I'll update it.

The only thing is that the default profile has a very different value from what we actually require.